From: raurodse <raurodse@gmail.com>
Date: Tue, 18 Jun 2019 13:24:33 +0200
Subject: disablethirdparty

---
 gui/gtk/stepPrepare.ui | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gui/gtk/stepPrepare.ui b/gui/gtk/stepPrepare.ui
index 0900a8f..0bf8fff 100644
--- a/gui/gtk/stepPrepare.ui
+++ b/gui/gtk/stepPrepare.ui
@@ -195,7 +195,7 @@
         </child>
         <child>
           <object class="GtkBox" id="box3">
-            <property name="visible">True</property>
+            <property name="visible">False</property>
             <property name="can_focus">False</property>
             <property name="orientation">vertical</property>
             <property name="spacing">6</property>
@@ -203,7 +203,7 @@
               <object class="GtkCheckButton" id="prepare_download_updates">
                 <property name="label" translatable="yes">Download updates while installing ${RELEASE}</property>
                 <property name="use_action_appearance">False</property>
-                <property name="visible">True</property>
+                <property name="visible">False</property>
                 <property name="can_focus">True</property>
                 <property name="receives_default">False</property>
                 <property name="xalign">0</property>
@@ -251,7 +251,7 @@
         </child>
         <child>
           <object class="GtkBox" id="nonfree_vbox">
-            <property name="visible">True</property>
+            <property name="visible">False</property>
             <property name="can_focus">False</property>
             <property name="orientation">vertical</property>
             <property name="spacing">6</property>
