From: raurodse <raurodse@gmail.com>
Date: Wed, 19 Jun 2019 22:28:34 +0200
Subject: llx_extra_packages

---
 scripts/plugininstall.py | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/scripts/plugininstall.py b/scripts/plugininstall.py
index dc82be4..c6954c6 100755
--- a/scripts/plugininstall.py
+++ b/scripts/plugininstall.py
@@ -101,6 +101,13 @@ class Install(install_misc.InstallBase):
                 for line in langpacks:
                     self.langpacks.append(line.strip())
 
+        # Lliurex Packages
+        self.lliurex_pkgs = []
+        if os.path.exists('/run/user/999/ubiquity/lliurex-extra-packages'):
+            with open('/run/user/999/ubiquity/lliurex-extra-packages') as llx_pkg:
+                for line in llx_pkg:
+                    self.lliurex_pkgs.append(line.strip())
+
         # Load plugins
         modules = plugin_manager.load_plugins()
         modules = plugin_manager.order_plugins(modules)
@@ -203,6 +210,16 @@ class Install(install_misc.InstallBase):
         except SystemError:
             pass
 
+        # Ignore failures from lliurex packages installation.
+        try:
+            self.do_install(self.lliurex_pkgs)
+        except install_misc.InstallStepError:
+            pass
+        except IOError:
+            pass
+        except SystemError:
+            pass
+
         self.next_region()
         self.remove_unusable_kernels()
 
