From: raurodse <raurodse@gmail.com>
Date: Mon, 15 Jul 2019 12:16:24 +0200
Subject: Set LliureX 19.07 on ubiquity

---
 ubiquity/frontend/kde_ui.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/ubiquity/frontend/kde_ui.py b/ubiquity/frontend/kde_ui.py
index 25d21cd..2946c2f 100644
--- a/ubiquity/frontend/kde_ui.py
+++ b/ubiquity/frontend/kde_ui.py
@@ -95,6 +95,9 @@ class UbiquityUI(QtWidgets.QMainWindow):
                             distro_release.endswith('"'):
                         distro_release = distro_release[1:-1]
 
+        distron_name = "LliureX"
+        distro_release = "19.07"
+
         self.distro_name_label.setText(distro_name)
         self.distro_release_label.setText(distro_release)
 
